pkg/plugins/golang/v3/scaffolds/webhook.go

```diff
@@ -90,6 +92,22 @@ func (s *webhookScaffolder) Scaffold() error {
 		return err
 	}
 
+	if doHubScaffold {
+		if err := scaffold.Execute(
+			&api.Conversion{Force: s.force, Hub: true},
```

Comment:
Could we not call only once &api.Conversion and pass doHubScaffold and doSpokeScaffold as argument for the constructor? 

`&api.Conversion{Force: s.force, Hub: true, DoHubScaffold: doHubScaffold,  DoSpokeScaffold: doSpokeScaffold},`

WDYT